src/Span.h

```diff
@@ -1375,4 +1431,206 @@ void Span::serialize_json(RapidJsonWriter &writer) const
     writer.EndObject();
 }
 
-} // namespace zipkin
\ No newline at end of file
+namespace __impl
+{
+
+struct Span2 {
+    enum Kind {
+        UNKNOWN,
+        CLIENT,
```

Comment:
My question is whether the client library need to support those broker message types?